docs/api/0.1.x/compute_resource.md

```diff
@@ -0,0 +1,52 @@
+# Compute resource API examples
+
+## list compute resources
+
+```clickhouse
+
+GET http://<foreman_url>:<port>/api/compute_resources
+
+```
+
+## Get info on specific compute resource
+```clickhouse
+GET http://<foreman_url>:<port>/api/compute_resources/<id>
+```
+
+## Create compute resource 
+```clickhouse
+
+POST http://<foreman_url>:<port>/api/compute_resources/
+
+body:
+{
+"compute_resource" : {
+     "name" : "<provider_name>",
+     "provider" : "KubeVirt",
+     "hostname": "<hostname_url>",
+     "token": "<token>" ,
+     "namespace" : "<name_space>", 
+     "api_port" : "<port>"
+  }
+}
+
+```
+
+
+## Update a compute resource  
+
+```
+PUT http://<foreman_url>:<port>/api/compute_resources/<id>
+
+body:
+{
+"compute_resource" : {
+     "name" : "<provider_name>",
+     "hostname": "<hostname_url>",
+     "token": "<token>" ,
+     "namespace" : "<name_space>", 
+     "api_port" : "<port>"
+  }
+}
+
+```
```

Comment:
What about examples for listing images / vms ? 
How to perform Import VM ? Associate/Disassociate VM ? Start / Stop / Delete VM ?
Not sure how common this are, but if that's easy to document, we might want to add them - unless they are generic for all compute resources and already covered by foreman API docs.